inc/classes/admin/settings/class-settings.php

```diff
@@ -376,7 +376,7 @@ public function sanitize_callback( $input ) {
 			'monthly' => 1,
 		];
 
-		$input['automatic_cleanup_frequency'] = isset( $cleanup_frequencies[ $input['automatic_cleanup_frequency'] ] ) ? $input['automatic_cleanup_frequency'] : $this->options->get( 'automatic_cleanup_frequency' );
+		$input['automatic_cleanup_frequency'] = isset( $input['automatic_cleanup_frequency'] ) && isset( $cleanup_frequencies[ $input['automatic_cleanup_frequency'] ] ) ? $input['automatic_cleanup_frequency'] : $this->options->get( 'automatic_cleanup_frequency' );
```

Comment:
Can be simpler: `isset( $input['automatic_cleanup_frequency'], $cleanup_frequencies[ $input['automatic_cleanup_frequency'] ] )`.